Python/sysmodule.c

```diff
@@ -2129,6 +2129,13 @@ static PyObject *
 sys__clear_type_cache_impl(PyObject *module)
 /*[clinic end generated code: output=20e48ca54a6f6971 input=127f3e04a8d9b555]*/
 {
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
+                     "sys._clear_type_cache() is deprecated and scheduled for removal in a "
```

Comment:
Sorry I didn't notice but can you keep the line length under 80 characters
 It seems it's a bit wider.